opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/SpanBuilderShimTest.java

```diff
@@ -296,6 +297,21 @@ void withStartTimestamp() {
     assertThat(spanData.getStartEpochNanos()).isEqualTo(micros * 1000L);
   }
 
+  @Test
+  void setAttribute_unrecognizedType() {
+    SpanShim span =
+        (SpanShim)
+            new SpanBuilderShim(telemetryInfo, SPAN_NAME).withTag("foo", BigInteger.TEN).start();
+    try {
+      SpanData spanData = ((ReadableSpan) span.getSpan()).toSpanData();
+      assertThat(spanData.getAttributes().size()).isEqualTo(1);
+      assertThat(spanData.getAttributes().get(AttributeKey.stringKey("foo")))
+          .isEqualTo(BigInteger.TEN.toString());
```

Comment:
Do you think there's any value in being explicit about the expected string? I.e. `.isEqualTo("10")`?

Reply:
Assertions now compare against literals directly ;)